[Keyboard] Add Quarkeys Z60, reorganized keyboard file directory (#19404)
qmk/qmk_firmware: keyboards/quarkeys/z60/readme.md

# Quarkeys Z60 Keyboard

> ![Z60](https://static.wixstatic.com/media/72b918_45f82258186646d4bfa57c9d73ccc9bc~mv2.jpeg)

Reviewer: Please rehost on Imgur.

Author: Hi drasma, I tried with Imgur but it has restriction for my region's cell number, so I guess I just delete the Picture in readme.md?

Author: Hi, probably stupid to ask, do I need to click the 'Resolve conversation' button?

# Quarkeys Z60 Keyboard

> ![Z60](https://static.wixstatic.com/media/72b918_45f82258186646d4bfa57c9d73ccc9bc~mv2.jpeg)
> ![Layout](https://static.wixstatic.com/media/72b918_bb37ec16543545b0aac331d5a7fa977c~mv2.png)

Reviewer 2: This image needs to be rehosted on Imgur.
```suggestion
![Z60](https://static.wixstatic.com/media/72b918_45f82258186646d4bfa57c9d73ccc9bc~mv2.jpeg)
```

A 60% Gasket keyboard with innovated top enclosure made by Quarkeys, second of out projects.

* Keyboard Maintainer: [Quarkeys Studio](www.quarkeys.com)
* Hardware Supported: Z60-ACR, Z60-Evolve
* Hardware Specs: Solder version with per-key RGBs, layer & Capslock indicators
                  Hotswap version with per-key RGBs, layer & Capslock indicators

Make example for this keyboard solder version keymap(after setting up your build environment):

    make quarkeys/z60/solder:default

> Make example for this keyboard hotswap version keymap:
> 
>     make quarkeys/z60/hotswap:default
> 

Reviewer 2: ```suggestion
```

Flashing example for this keyboard:

    make quarkeys/z60/solder:default:flash

> **Bootloader:** Press the physical reset button (ECS position of layer 1)

Reviewer 2: ```suggestion
**Bootloader:** Press the `QK_BOOT` keycode at ESC position of layer 1.
```
